.travis.yml

```diff
@@ -5,6 +5,8 @@ go:
   - 1.12.x
   - tip
 
+go_import_path: github.com/onsi/ginkgo
```

Comment:
In the future someone might remove this line saying that it's unnecessary, which is true for the original. I wonder whether it's worth adding a comment to say that this has been added to the original as a convenience for forks?

Reply:
Yes, good idea.
Another option to allow forked repo to pass travis-ci, is to add this change to the forkrd repo only, but that will not be permanent, once the fork is reset to match upstream.